addon/templates/components/bread-crumb.hbs

```diff
@@ -1,15 +1,31 @@
 {{#if route.linkable}}
-  {{#link-to route.path class=linkClass}}
-    {{#if hasBlock}}
-      {{yield this route}}
-    {{else}}
-      {{route.title}}
-    {{/if}}
+  {{#link-to route.path class=linkClass itemscope=true itemtype="http://schema.org/Thing" itemprop="item"}}
```

Comment:
FYI These attrs it the `link-to` with not be compiled if we don't reopen the `LinkComponent` 

Have you thought about using the `href-to` [addon](https://github.com/intercom/ember-href-to) here?